spec/javascripts/utils/timezoneHelper.spec.js

```diff
@@ -0,0 +1,50 @@
+import expect from 'expect';
+import formatDate from 'src/utilities/timezoneHelper';
+
+describe('formatDate', () => {
+  it('should correctly format the date', () => {
+    const input = '23.06.2023, 12:00:00';
+    const expectedOutput = 'Friday, June 23, 2023 at 12:00 PM';
+
+    const result = formatDate(input);
+
+    expect(result).toEqual(expectedOutput);
+  });
+
+  it('should handle different date formats', () => {
```

Comment:
Isnt this the same test as the one before just with a different date but the same format?

Reply:
deleted the second one. I was testing with different dates